src/main/java/frc/robot/RobotContainer.java

```diff
@@ -177,46 +213,63 @@ private void initCommands() {
   // it to a {@link
   // edu.wpi.first.wpilibj2.command.button.JoystickButton}.
   private void configureButtonBindings() {
+    controllerCheck();
 
     // Intake
     if (intakeForwardCommand != null && intakeReverseCommand != null) {
       buttons[Constants.RBumper].whileHeld(intakeForwardCommand);
       buttons[Constants.RTriggerButton].whileHeld(intakeReverseCommand);
     }
 
-    // Shooter
-    if (shooterSubsystem != null && shooterHeld != null) {
-      buttons[Constants.backButton].whenPressed(shooterHeld);
-      buttons[Constants.LJoystickButton].whenPressed(
-          new InstantCommand(shooterSubsystem::cycleAimModeNext, shooterSubsystem));
-      buttons[Constants.RJoystickButton].whenPressed(
-          new InstantCommand(shooterSubsystem::cycleAimModePrevious, shooterSubsystem));
-    }
+    if (axisCount1 == 0 && buttonCount1 == 0) {
+      
+      // Shooter
+      if (shooterSubsystem != null && shooterHeld != null) {
+        buttons[Constants.backButton].whenPressed(shooterHeld);
+        buttons[Constants.LJoystickButton].whenPressed(
+            new InstantCommand(shooterSubsystem::cycleAimModeNext, shooterSubsystem));
+        buttons[Constants.RJoystickButton].whenPressed(
+            new InstantCommand(shooterSubsystem::cycleAimModePrevious, shooterSubsystem));
+      }
 
-    //CDS
-    if (CDSSubsystem != null) {
-      CDSForwardCommand = new CDSForwardCommand(CDSSubsystem);
-      CDSSubsystem.setDefaultCommand(new CDSAutoAdvanceCommand(CDSSubsystem));
-      //CDSReverseCommand = new CDSReverseCommand(CDSSubsystem, shooterSubsystem);
-      //CDSSubsystem.senseColor();
-    }
-    
-    if (CDSForwardCommand != null && outtakeCommand != null) {
-      buttons[Constants.LTriggerButton].whileHeld(CDSForwardCommand);
-      //buttons[Constants.RTriggerButton].whileHeld(CDSReverseCommand);
-      buttons[Constants.RTriggerButton].whileHeld(outtakeCommand);
-    }
+      // CDS
+      if (CDSSubsystem != null) {
+        CDSForwardCommand = new CDSForwardCommand(CDSSubsystem);
+        CDSSubsystem.setDefaultCommand(new CDSAutoAdvanceCommand(CDSSubsystem));
+      }
 
-    // Limelight
-    if (limelightAlign != null) {
-      buttons[Constants.startButton].whenPressed(limelightAlign);
-    }
+      if (CDSForwardCommand != null && outtakeCommand != null) {
+        buttons[Constants.LTriggerButton].whileHeld(CDSForwardCommand);
```

Comment:
Left trigger should be while held shoot with out lime light auto aim. So the command for the fender shot.

Reply:
Even just for testing mode?

Reply:
I thought that the shooter team wanted that for competition but for testing, they wanted it more manual.